backbone.js

```diff
@@ -76,6 +79,10 @@
   //
   // `Function#apply` can be slow so we use the method's arg count, if we know it.
   var addMethod = function(length, method, attribute) {
+    // In case we don't catch it, deopt Underscore functions that contain
+    // the arguments keyword.
+    if (length > 0 && containsArguments.test(_[method])) length = 0;
```

Comment:
Scratch that, I thought `#test` was included in http://www.2ality.com/2015/09/well-known-symbols-es6.html#forwarding_calls_from_string_methods_to_their_parameters.